docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
 
     # chkconfig mysqld on
 
-Create Database
+Create database
 
     # mysqladmin -uroot create vnet
     # cd /opt/axsh/openvnet/vnet
     # bundle exec rake db:init
 
+To insert the database records use `vnctl`
+
+```
+# cd /opt/axsh/openvnet/vnctl
+# bundle install --path vendor/bundle
+```
 
-Start the OpenVNet services
----------------------------
+Start vnmgr and webapi since `vnctl` is just a webapi client.
 
     # initctl start vnet-vnmgr
     # initctl start vnet-webapi
-    # initctl start vnet-vna
-
-OpenVNet writes its logs in the /var/log/openvnet directory. If there's a problem starting any of the services, you can find its log files there.
-
-
-
----------------------------------------------------------------------
-
-# Let's try 1Box OpenVNet#
 
-This section explains how to install OpenVNet on one node. This is a great way to try it out.
-![1BOX](OpenVNet_1Box_environment.gif)
+From now on you need to create a couple of database records to launch your OpenVNet environment.
 
-Install the following operation system and kernel.
+#### Datapath
 
-+ RHEL (or Centos) 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
+```
+# ./bin/vnctl datapaths add --uuid dp-test1 --display-name test1 --dpid 0x0000aaaaaaaaaaaa --node-id vna
+```
 
+* dpid
 
-Installation and Setup of OpenVNet
------------------------------------
+The datapath ID specified in `/etc/sysconfig/network-scripts/ifcfg-br0`
 
-### Network interfaces setup
+* node-id
 
-We're going to enslave eth0 to openvswitch so OpenVNet can use it. You will no longer be able to use eth0 to ssh into the node or access the Internet. Prepare another interface for that if you need to.
+The ID of the vna written in `/etc/openvnet/vna.conf`
 
-    # vi /etc/sysconfig/network-scripts/ifcfg-eth0
-    DEVICE=eth0
-    DEVICETYPE=ovs
-    TYPE=OVSPort
-    OVS_BRIDGE=br0
-    BOOTPROTO=none
-    ONBOOT=yes
-    HOTPLUG=no
-    #
-    # vi /etc/sysconfig/network-scripts/ifcfg-br0
-    DEVICE=br0
-    DEVICETYPE=ovs
-    TYPE=OVSBridge
-    ONBOOT=yes
-    BOOTPROTO=static
-    IPADDR=172.16.20.41
-    NETMASK=255.255.255.0
-    HOTPLUG=no
-    OVS_EXTRA="
-     set bridge     ${DEVICE} protocols=OpenFlow10,OpenFlow12,OpenFlow13 --
-     set bridge     ${DEVICE} other_config:disable-in-band=true --
-     set bridge     ${DEVICE} other-config:datapath-id=00004e6d2b508f4c --
-     set bridge     ${DEVICE} other-config:hwaddr=02:01:00:00:00:01 --
-     set-fail-mode  ${DEVICE} standalone --
-     set-controller ${DEVICE} tcp:127.0.0.1:6633
-    "
 
-### OpenVNet package installation
+#### Network
 
-Please install OpenVNet according to the "Pre-setup" and "OpenVNet installation" sections of "Preliminary Operations and Installation".
+```
+# ./vnctl networks add --uuid nw-test1 --display-name testnet1 --ipv4-network 10.100.0.0 --ipv4-prefix 24 --network-mode virtual
```

Comment:
Just now we were calling `bin/vnctl` but now we are calling `./vnctl` without telling the reader to change directories. Anyway, once the vnctl RPM is merged, the working directory will no longer matter.